### DISCUSSION TITLE
Next State in a SM

### DISCUSSION THREAD

**Discussion by ValeMP** [Cotalker/documentation#350] [2. Questions and Answers] [answered]
Estoy creando una maquina de estado en el workflow llamado "manejo de soluciones" en Cotalker, y resulta que en un estado que tiene dos posibles estados, solo se visualiza uno. No encuentro bien la razón de por qué sucede esto, por si alguno me podría ayudar.

Gracias!

**Comment by guillevalin** [comment]
Se revisó con @ValeMP y la aplicación se comporta según lo esperado.

**Reply by ValeMP** [reply]
Me volvió a suceder, pero no sé si es un comportamiento esperado en alguna circunstancia. 
Dejaré antecedente de lo que realicé.

1. cree una tarea en la padre. 
2. En el estado de "demo en desarollo" cree uan tarea hija.
3. Llegue al estado de "preset en desarrollo".
4. No se visualizaba el cambio de estado.
5. Refresh de la pagina, pero aún no se veía  el cambio de estado.
6. Cambie de estado a la tarea hija. ( a un estado "end")
7. Se visualizó la opción de cambio de estado. 

Despues de eso, la máquina funcionó como esperado. 

Lo dejó escrito aquí por si sirve para un futuro

**Comment by ValeMP** [comment, marked as answer]
Al final, era un comportamiento esperado. Si no estan todas las tareas hijas resueltas, no se puede llegara a un estado final de la SM padre.

Pero de igual modo, la tarea padre se puede mover en los estados "In-progress".
